test/v1/connectors/CreateAndAmendEmploymentExpensesConnectorSpec.scala

```diff
@@ -40,29 +40,57 @@ class CreateAndAmendEmploymentExpensesConnectorSpec extends ConnectorSpec {
     )
   )
 
-  trait Test { _: ConnectorTest =>
+  trait Test {
+    _: ConnectorTest =>
+
+    def taxYear: TaxYear
 
     val connector: CreateAndAmendEmploymentExpensesConnector = new CreateAndAmendEmploymentExpensesConnector(
       http = mockHttpClient,
       appConfig = mockAppConfig
     )
 
+    val request: CreateAndAmendEmploymentExpensesRequest = CreateAndAmendEmploymentExpensesRequest(Nino(nino), taxYear, body)
+
   }
 
-  "amend" should {
-    val request = CreateAndAmendEmploymentExpensesRequest(Nino(nino), taxYear, body)
+  "CreateAmendEmploymentExpensesConnector" when {
+
+    "amend" should {
+
+      "put a body and return 204 no body" in new IfsR6Test with Test {
+
+        def taxYear: TaxYear = TaxYear.fromMtd("2021-22")
+
+        val outcome = Right(ResponseWrapper(correlationId, ()))
+
+        willPut(
+          url = s"$baseUrl/income-tax/expenses/employments/$nino/2021-22",
+          body = body
+        )
+          .returns(Future.successful(outcome))
 
-    "put a body and return 204 no body" in new IfsR6Test with Test {
-      val outcome = Right(ResponseWrapper(correlationId, ()))
+        await(connector.createAmendEmploymentExpenses(request)) shouldBe outcome
```

Comment:
Can the result be assigned to a `val result`? (& rename outcome). Next line is then `result shouldBe expectedOutcome`